### DISCUSSION TITLE
GPU fans wont stop

### DISCUSSION THREAD

**Discussion by iafrate97** [Rem0o/FanControl.Releases#733] [Q&A] [answered]
I have a 3070 ti and once the fans start spinning they will not stop. It goes to 30% as the lowest, even if i put in manual the lowest it goes is 30% but does control it above 30. i've tried having the threshold at 30 and still doesn't stop it. The only thing that stops the fans is if i refresh but once they start they don't stop again.

**Comment by Rem0o** [comment, marked as answer]
That's a known limitation, not much we can do until the API is available to trigger the fan stop.

**Reply by iafrate97** [reply]
Alright as long as its not something i'm doing, thank you

**Reply by MadGoatHaz** [reply]
Curious how MSi AfterBurner implements this into their fan control then?

**Comment by iPodge** [comment]
It now works for me with the 114 release.

**Reply by iafrate97** [reply]
even on 115 it still does not work for me.

**Reply by MisterMisty** [reply]
What card are you using?

**Comment by ggondra** [comment]
나도 이 문제가 있었지만 듀얼모니터 이상의 사용환경에서 문제가 발생합니다. 일시적으로 싱글모니터로 세팅하면 속도는 0이 됩니다. 하지만 게임을 플레이하고나면 다시 30%가 됩니다. 이것은 일부 그래픽카드들의 특징입니다.

**Reply by iafrate97** [reply]
네, 저는 3개의 모니터를 가지고 있지만 하나로 전환해도 아무 것도 변경되지 않습니다.

**Comment by nigiriemoji** [comment]
I have the same issue with a MSI 3070 card, 30% min

**Comment by MisterMisty** [comment]
Could Fan Control force switch a fan control off and then on again, that way resetting it so the fan stops?
Automating the task away from the user having to turn the fan control off and on each time.

**Reply by Rem0o** [reply]
That's a workaround specific to some Nvidia cards. Also won't be reliable, since you "expect" the fans to go 0 RPM when turning off, but that's not a guarantee, and I will have 10 emails in my inbox next morning telling me something weird is happening when they set 0 %.

**Reply by MisterMisty** [reply]
Thanks for the explanation Rem0o, makes complete sense. Amazing work with fan control
